govcd/tm_region.go

```diff
@@ -33,6 +33,16 @@ func (vcdClient *VCDClient) CreateRegion(config *types.Region) (*Region, error)
 	return createOuterEntity(&vcdClient.Client, outerType, c, config)
 }
 
+// CreateRegionAsync creates a new region and returns its tracking task
+func (vcdClient *VCDClient) CreateRegionAsync(config *types.Region) (*Task, error) {
```

Comment:
Just one question: what for do we use the previous CreateRegion(...) function after this change?

Reply:
We don't directly use it in here anymore.